pytorch/training/docker/1.6.0/py3/cu110/Dockerfile.gpu

```diff
@@ -15,7 +15,7 @@ ARG RMM_VERSION=0.15.0
 
 # The smdebug pipeline relies for following format to perform string replace and trigger DLC pipeline for validating
 # the nightly builds. Therefore, while updating the smdebug version, please ensure that the format is not disturbed.
-ARG SMDEBUG_VERSION=1.0.0
+ARG SMDEBUG_VERSION=1.0.5
```

Comment:
Please submit a different PR for PT 1.6, since the CI can only build one framework at a time, and it is currently targeted at PT 1.8.